File: Spring-boot-JWT-with-angular-Part-1-main/src/main/java/com/jwt/secqurity/sales/Entity/Sales.java
package com.jwt.secqurity.sales.Entity;

import com.jwt.secqurity.purchase.entity.PurchaseBase;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.Entity;

@Entity
@Getter
@Setter
@ToString
public class Sales extends SalesBaseModel{
    private String  customer_name;
    private String catagory;
    private String sub_catagory;
    private String product_name;
    private String brand_name;
    private String sku;
    private String supplier_name;
    private String status;
    private Long qty;
    private String unit;
    private Long selling_price;
    private Long discount_percentage;
    private Long tax_percentage;
    private Long total;
    private Long paid;
    private Long due;
    private String payment_status;
    private String date;
}

File: SalesAndInventory/src/app/sales/service/sales.ts
export interface Sales{
   
    sales_id: number;
    customer_name:string;
    catagory:string;
    sub_catagory:string;
    product_name: string;
    brand_name: string;
    sku:string;
    supplier_name: string;
    status: string;
    qty: number;
    unit: string;
    selling_price: number;
    tax_percentage:number;
    discount_percentage:number;
    total: number;
    paid: number;
    due:number;
    payment_status:string;
    date:Date;
    }